src/modules/integrator-guide/pages/automation/workflows/examples/template-rendering.adoc

```diff
@@ -0,0 +1,164 @@
+= Template rendering
+:attachment-path: ../../_attachments/workflow/
+
+In this example, we'll take a look at how to render different templates that were made in our xref:templates/index.adoc[template system].
+
+To start, create a template of type HTML in our templating system located in the admin area.
+
+== HTML
+
+Create a workflow similar to the one below. Make sure to look at the step details of steps (2) and (3).
+
+.The screenshot outlines the workflow that renders an HTML template and sends it as an email.
+image::workflow/example-template-rendering/html-workflow.png[role="data-zoomable"]
+
+The link:{attachment-path}template_rendering_html.json[source code] for the workflow example.
```

Comment:
Please reword to `The source code for the workflow can be found here.` where `here` is the link (else you'll get chewed up by the reviewer as I have)

Reply:
This was coppied from here, so change it there too please, so someone doesn't make the same mistake as me
https://docs.cortezaproject.org/corteza-docs/2021.9-develop/integrator-guide/automation/workflows/examples/email-attachment.html